### PR TITLE
🔥 class-methods-use-this

### PR DESCRIPTION
**Pull request by davidcornu**
https://github.com/Shopify/javascript/issues/263

### CODE REVIEW

**Comment by GoodForOneFare** [comment]
FYI, Chris is on vacation until the 17th.

**Comment by davidcornu** [comment]
@GoodForOneFare am not in any particular hurry 😛

File: lib/config/rules/best-practices.js [hunk 1]
```diff
@@ -7,8 +7,6 @@ module.exports = {
   'array-callback-return': 'error',
   // Treat var statements as if they were block scoped
   'block-scoped-var': 'warn',
-  // Enforce that class methods utilize this
-  'class-methods-use-this': 'warn',
```
Comment (lemonmade) [inline lib/config/rules/best-practices.js]: Can we please keep the rule but just set it to `off`?

**Review by lemonmade — APPROVED** [review]
Just switch to explicitly off versus excluding the rule, then we're good to go 👍

File: lib/config/rules/best-practices.js (hunk 1 above)
Comment (davidcornu) [reply lib/config/rules/best-practices.js]: Sure thing

**Comment by GoodForOneFare** [comment]
Please update the CHANGELOG, too.

**Comment by davidcornu** [comment]
Changed to `off` and updated CHANGELOG.

**Pull request merged**